build.proj

```diff
@@ -78,6 +78,63 @@
     <ItemGroup>
       <ReferenceOnlyPackageDlls Remove="$(ReferencePackagesSourceDir)**/System.Runtime.dll" />
       <ReferenceOnlyPackageDlls Remove="$(ReferencePackagesSourceDir)**/netstandard.dll" />
+      <!-- Microsoft.AspNet.App DLLs that include .resource files -->
```

Comment:
I can't see why these would be in the list of ReferenceOnlyPackageDlls.  I spot-checked a number of them and they all seem like normal packages.

Reply:
Microsoft.AspNet.App is recognized as a reference package because it only has `_._` in its lib folder, but it includes a few dozen full assemblies in a `ref` folder that cause issues when we try to ildasm/ilasm them.  I'll reach out to ASP.Net and see if that should actually be happening.

Reply:
Is it all dlls in Microsoft.AspNet.App?  If so, this could be shortened to 
`<ReferenceOnlyPackageDlls Remove="$(ReferencePackagesSourceDir)microsoft.aspnet.app/*" />`
to remove all of them.

Reply:
Changed as suggested.

Reply:
Looks like these also end up in other directories - I had to revert this change.